old_py2/controllers/admin/admin_cron_controller.py

```diff
@@ -237,16 +237,13 @@ class AdminBackfillPlayoffTypeDo(LoggedInHandler):
     Set playoff types
     """
 
-    # These offseasons played the 2014 game
-    EXCEPTIONS_2015 = ['2015cc', '2015cacc', '2015mttd']
-
     def get(self, year):
         self._require_admin()
         year = int(year)
         events = EventListQuery(year).fetch()
         for event in events:
             if not event.playoff_type:
-                if event.year == 2015 and event.key_name not in self.EXCEPTIONS_2015:
+                if event.game_year == 2015:
```

Comment:
This is no longer right - but also, I'm PRETTY SURE we don't need the `AdminBackfillPlayoffTypeDo` command anymore